engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -80,44 +80,8 @@ def _init_model(self, kwargs, model_id_or_path):
             self._warmup(**kwargs)
 
     def _warmup(self, **kwargs):
-        max_prefill_tokens = int(
-            self.properties.get(
-                "max_rolling_batch_prefill_tokens",
-                int(self.properties.get("max_rolling_batch_size", 4)) * 272))
-
-        # Todo: Find the maximum input_length dynamically
-        input_length = 1024
-
-        n_tokens = 0
-        req_id = 0
-        requests = []
-
-        while n_tokens < max_prefill_tokens:
-            truncate = min(input_length, max_prefill_tokens - n_tokens)
-            requests.append(
-                lmi_dist.utils.types.Request(
-                    id=req_id,
-                    inputs='_test ' * input_length,
-                    parameters=NextTokenChooserParameters(
-                        temperature=0.9,
-                        repetition_penalty=1.2,
-                        top_k=10,
-                        top_p=0.9,
-                        typical_p=0.9,
-                        do_sample=False,
-                        seed=0),
-                    stopping_parameters=StoppingCriteriaParameters(
-                        stop_sequences=[], max_new_tokens=2),
-                    truncate=truncate,
-                    prefill_logprobs=True))
-            n_tokens += input_length
-            req_id += 1
-
-        batch = self.batch_cls.get_batch(
-            Batch(id=0, requests=requests,
-                  size=len(requests)), self.model.tokenizer,
-            kwargs.get("torch_dtype", torch.float16), self.device)
-        self.model.warmup(batch)
+        batch_size = int(self.properties.get("max_rolling_batch_size", 4))
```

Comment:
the default max_rolling_batch_size in java side is 32.

Reply:
Changed